Assets/Runtime/Window/WindowUILayer.cs

```diff
@@ -110,6 +110,11 @@ public class WindowUILayer : AUILayer<IWindowController>
             base.ReparentScreen(controller, screenTransform);
         }
 
+        public bool IsWindowOpened(string screenId)
+        {
+            return registeredScreens.TryGetValue(screenId, out var ctl) && CurrentWindow == ctl;
```

Comment:
ctl is not clear.

Reply:
@eliatlas, agree, but I've used the abbreviator that uiframework is using
let keep it align otherwise it will be even worse